lms/djangoapps/course_home_api/progress/v1/tests/test_views.py

```diff
@@ -97,3 +102,14 @@ def test_masquerade(self):
         # Masquerade as verified user
         self.update_masquerade(username=verified_user.username)
         assert self.client.get(self.url).data.get('enrollment_mode') == 'verified'
+
```

Comment:
Should there be tests for the other new fields you introduced as well? Doesn't need to be extensive, but seems like we'd want to verify

Reply:
sure, added a test for both end and user_has_passing_grade

Reply:
Ah sorry, I should have been more clear here. It might be easier to just wrap up these checks (and maybe all three) into an existing test. We already have happy path tests and could just add in the small changes you might want there rather than do the setup of new tests. 

This is all a nit. The tests as they are are totally fine too.